package.json

```diff
@@ -8,7 +8,9 @@
   "scripts": {
     "lint": "eslint --ext .js,.ts,.vue ./",
     "format": "prettier --write \"**/*.{js,ts,vue,scss,html,md,json}\" --ignore-path .gitignore",
-    "test": "echo \"No test specified\" && exit 0"
+    "test": "echo \"No test specified\" && exit 0",
```

Comment:
If you don't use jest, I rather prefer to see eslint and prettier here as a sort of "test" if code looks good

Reply:
Done, it works fine. Thanks for the suggestion!